### DISCUSSION TITLE
Update Zimwiki entry data on wikimatrix.org

### DISCUSSION THREAD

**Discussion by Daksol** [zim-desktop-wiki/zim-desktop-wiki#2600] [General]
I was involved in selecting a wiki for a voluntary organisation, and one resource we found very useful was https://www.wikimatrix.org

This site allows you to do a side by side comparison of features of (say) 3 or 4 different wikis - drawn from a pool of 82 options (as per April 2024).

Zimwiki is included as one of the 82 options.

This URL will give you a side-by-side comparison of Zimwiki, Wikidot and Tiddlywiki
https://www.wikimatrix.org/compare/wikidot+tiddlywiki+zim

I noticed that some of the information on the site is out of data - for instance "last release of product". It indicates that Zimwiki's entry there not updated for over ten years.

Could i suggest that the entry there is updated - based on that site now, Zimwiki looks like a moribund product - and that is definitely not true!

Best regards
Image shows data you will see at: https://www.wikimatrix.org/compare/zim

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/42010857/2e9f62bc-084f-48a5-b938-024060f8b8e7)

https://www.wikimatrix.org/compare/zim+foswiki+wiki-js

**Comment by dehesselle** [comment]
You need to talk to the company that runs that page. We're not affiliated in any way with them and are not responsible for them keeping their data up to date. We're also not going to do it for them - they are a company with commercial interests, we're not.

**Reply by Daksol** [reply]
OK. I thought that the matrixwiki site is a valid mechanism by which Zimwiki might get new customers - which is good reinforcement and encouragement for non-commercial organisations too. Hence some small effort to update would be worthwhile.

Looking at how entries get added, I expect that someone from the Zimwiki team provided the info to them originally.

**Reply by introt** [reply]
To clear some things up, "we" are not a non-commercial/non-profit organization - that implies an organized entity - but a rag-tag group of volunteers helping Jaap maintain the software. Personally, I don't think we have any customers either, I just think Jaap uses that term by the way of habit in some of the docs.

**Reply by dehesselle** [reply]
> Looking at how entries get added, I expect that someone from the Zimwiki team provided the info to them originally.

No, that's certainly not the case.

**Comment by Daksol** [comment]
Thanks
